### ISSUE TITLE
Markers cannot be moved using touch screen

### ISSUE THREAD

**Issue by wvankoppen** [geoman-io/leaflet-geoman#1248]
When I run _demo/events.html_ on my mobile phone, it is not possible to move markers to a different location on the map via touch screen. I got the following error in the browser:

```
LatLng.js:32 Uncaught Error: Invalid LatLng object: (NaN, NaN)
    at new v (LatLng.js:32:9)
    at Object.unproject (Projection.SphericalMercator.js:35:10)
    at Object.pointToLatLng (CRS.js:41:26)
    at e.unproject (Map.js:989:27)
    at e.layerPointToLatLng (Map.js:997:15)
    at e.containerPointToLatLng (Map.js:1054:15)
    at e._overwriteEventIfItComesFromMarker (Dragging.js?f8e5:406:1)
    at e._dragMixinOnMouseDown (Dragging.js?f8e5:165:1)
    at e._simulateMouseDownEvent (Dragging.js?f8e5:125:1)
    at HTMLImageElement.o (DomEvent.js:108:13)
```

Note that moving rectangles or polygons work fine, even on mobile phone using touch screen.
On my MacBook using touchpad both moving rectangles and markers work fine.

**Expected result:**
- Both rectangles and markers can be moved on the map using touch screen.

**Actual result:**
- Markers cannot be moved on the map using touch screen.

**Comment by Falke-Design** [comment]
Thank you for letting us know. Looks like it is the same as #1208

**Comment by Falke-Design** [comment]
@wvankoppen I can't reproduce the error in Safari & Mac. Can you please share your data and confirm that it is also happening in the demo https://jsfiddle.net/o1dwu2vg/ ?

**Comment by wvankoppen** [comment]
@Falke-Design Thanks for the quick reply!

I checked your demo and indeed it is working correctly in this example.
Surprised by what could be the cause why it did not work in my initial example, I found out that the drag behavior broke down on mobile phone using Leaflet 1.8.0 and higher. In Leaflet version 1.6.0 (used in the example) and also 1.7.0 it works fine.

Could you check this example using leaflet 1.8.0 or higher?

**Comment by Falke-Design** [comment]
Can you please share your IOS and safari version?

Here is a updated version, can you please test again: https://jsfiddle.net/e12utc5r/

**Comment by wvankoppen** [comment]
@Falke-Design I'm using a Motorola phone (running Android 11) so not an iPhone.
I verified the new demo and here dragging does not work either.
What are the differences between the two demos?

In order to provide logging, I need to connect to my phone using USB to attach the debugger (and currently no cable here) so I can add logging later.

**Comment by Falke-Design** [comment]
I switched to Leaflet 1.9.2 in the Demo.

For now I don't need a log. I will send you in around 4 hours a link with the newest version (including the fix) and if you can test it again, would it be great

**Comment by wvankoppen** [comment]
perfect! Looking forward to the fix, let me know and I'll retest it.

**Comment by Falke-Design** [comment]
@wvankoppen can you please test:

Failing: https://florian-bischof.at/tests/geoman-old/demo/events.html

Working: https://florian-bischof.at/tests/geoman/demo/events.html

**Comment by wvankoppen** [comment]
@Falke-Design  I tested both links, and can confirm that the demo on the second link is working also on mobile (First link is indeed failing on mobile).

So, nice job! How did you tackle the issue?

**Comment by Falke-Design** [comment]
@wvankoppen great to hear!

> How did you tackle the issue?

I didn't 😄 I just hoped that #1208 is fixing it ^^

**Comment by wvankoppen** [comment]
seems that you were right :)
When will the fix for #1208 be released?
